main/webapp/modules/core/scripts/index/default-importing-controller/controller.js

```diff
@@ -235,9 +235,9 @@ Refine.DefaultImportingController.prototype.updateFormatAndOptions = function(op
             if(finallyCallBack){
               finallyCallBack();
             }
-        } else {
+        } 
             callback(o);
-        }
+        
```

Comment:
@WaltonG: there is a POST asynch request here. Can you put the .fail() error handler? For the error message, a generic API network error (created it if it doesn't exist) seams enough to me. What do you think?

Reply:
I don't know how I overlooked it having  handled a similar issue earlier.